mmv1/templates/terraform/examples/scc_v2_project_mute_config_basic.tf.erb

```diff
@@ -0,0 +1,12 @@
+resource "google_project" "default" {
+  name       = "My Test Project"
```

Comment:
The parent of this project need to be specified and should be the test org.

Reply:
addressed